test/test_core.py

```diff
@@ -5900,6 +5900,16 @@ def test_fs_rename_on_existing(self, args):
       self.set_setting('FORCE_FILESYSTEM')
     self.do_runf('fs/test_fs_rename_on_existing.c', 'success', emcc_args=args)
 
+  @parameterized({
+    '': ([],),
+    'nodefs': (['-DNODEFS', '-lnodefs.js'],),
+    'noderawfs': (['-sNODERAWFS'],)
+  })
+  def test_fs_mkdir_dotdot(self, args):
+    if self.get_setting('WASMFS'):
+      self.set_setting('FORCE_FILESYSTEM')
```

Comment:
Are you running this test in the wasmfs configuration?  If not, and these two lines are just cargo-culted, I think we can just remove them.

Reply:
This is definitely cargo culted. Seems like it could be a good idea to run the test in wasmfs though -- I'm not sure how that is determined.

Reply:
We have a `wasmfs` mode in which you can run all the tests.  I believe these lines relate to that, but unless you are actually using that mode I think you can just skip these lines.

Reply:
Doesn't the wasmfs mode run in CI?

Reply:
No, not as far as I know